src/api_wrappers/wrapper_table.json

```diff
@@ -534,6 +534,15 @@
             "wikiLink": "https://wiki.dnanexus.com/API-Specification-v1.0.0/Visibility#API-method%3A-%2Fclass-xxxx%2FsetVisibility"
         }
     ],
+    [
+        "/database-xxxx/showFiles",
+        "databaseShowFiles(req, objectId)",
+        {
+            "objectMethod": true,
+            "retryable": true,
+            "wikiLink": "https://wiki.dnanexus.com/API-Specification-v1.0.0/Tags#API-method%3A-%2Fclass-xxxx%2FshowFiles"
```

Comment:
This link is wrong. It will be more like `"https://wiki.dnanexus.com/API-Specification-v1.0.0/Databases#API-method%3A-%2Fdatabase-xxxx%2FshowFiles"` once you put it to dev-wiki